src/components/assessment/AssessmentWorkspace.tsx

```diff
@@ -21,6 +24,7 @@ import Workspace, { WorkspaceProps } from '../workspace';
 import { ControlBarProps } from '../workspace/ControlBar';
 import { SideContentProps } from '../workspace/side-content';
 import Autograder from '../workspace/side-content/Autograder';
+
```

Comment:
What are all these spaces? Can we remove them so that tslint can properly apply the rules, or group them in a way that it makes sense.

e.g.
```javascript
import ... from '../../utils/...';
import ... from '../../utils/...';
import ... from '../../utils/...';

import ... from '../workspace/...';
import ... from '../workspace/...';
import ... from '../workspace/...';
```